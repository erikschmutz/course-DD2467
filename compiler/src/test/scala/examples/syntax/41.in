type 'a generic = {
  left: a
};;