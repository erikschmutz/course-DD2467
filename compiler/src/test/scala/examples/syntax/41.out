Program(
    List(
        TypeDeclaration(
            generic,
            TypeFieldDeclarations(
                List(
                    TypeFieldDeclaration(
                        left,TypePrimitive(a)
                    )
                )
            ),
            List(
                TypeOption('a)
            )
        )
))